### ISSUE TITLE
Residential evictions executed by city marshals

### ISSUE THREAD

**Issue by AmandaDoyle** [NYCPlanning/db-equitable-development-tool#51]
Description: Number of residential evictions executed by marshal

Logic: 
- Assign PUMA to each record
- Sum the number of evictions that have happened since Jan 1, 2017 (earliest eviction is Jan 3, 2017)

Source(s): https://data.cityofnewyork.us/City-Government/Evictions/6z8x-wfk4

Year(s):
- [ ] 2017-present

Geographies:
- [ ] PUMA
- [ ] Borough
- [ ] City

Race and ethnicity breakdown: No
Denominator:

SME: HPD

Questions:

Roadblocks:

**Comment by SashaWeinstein** [comment]
Around 90% of the evictions have a lat/long and can be assigned to a PUMA without geocoding. The evictions without coordinates seem to be mostly misspellings and records with addresses like "1013 EAST 96TH STREET - BASEMENT-REAR ROOM"
Should we try to geocode these 10% of records? Around 6.3k evictions

**Comment by AmandaDoyle** [comment]
Yes, if it's low hanging fruit we should try to geocode the records and increase the percentage of records with lat/longs and/or add PUMA info from GeoSupport

**Comment by SashaWeinstein** [comment]
Geocoding is documented in issue #63. 

I don't know if writing tests for this indicator really makes sense as I don't know how to run the tests in a container in github actions. If unit tests are written here are two I think could be helpful:
1. Ensure the PUMA assigned from the geocoding matches the borough of the eviction
2. Make sure that total evictions citywide equals sum of evictions across all boroughs. Assume each eviction gets a borough, we know this not to be true for PUMAs

**Comment by SashaWeinstein** [comment]
Hey @AmandaDoyle is `evictions` a suitable indicator label? `evictionscitymarshals` seems a little wordy. We also have an eviction cases filed indicator but that indicator has "filings" in the label

**Comment by AmandaDoyle** [comment]
@levysamu  we aren't calculating a rate.  This is just a count.

**Comment by levysamu** [comment]
Ok, so we don't have the information to populate the below screenshot?

![image](https://user-images.githubusercontent.com/17015121/161307143-876b0df7-aeec-436a-8bc0-e0aeb804e7a9.png)

**Comment by AmandaDoyle** [comment]
Correct.
